GovUkDesignSystem/Helpers/HtmlGenerationHelpers.cs

```
@@ -152,9 +152,10 @@ public static void SetErrorMessages(IHasErrorMessage target, ModelStateEntry mod
                 {
                     values.Add((string)modelStateEntry.RawValue);
                 }
-                return bool.Parse(values
```

This was just broken, when we have just the dummy value Single() will fail.